stack-deploy/src/types.rs

```diff
@@ -585,7 +585,7 @@ impl From<&TemplateBody> for String {
 
 impl From<&str> for TemplateBody {
     fn from(value: &str) -> Self {
-        value.to_string().into()
+        Self(value.into())
```

Comment:
__comment:__ TBH I am not 100% sure about this one. The other is very straight forward, and both could even be optimized down to the same thing.

Reply:
If there is a `From<String>` instance we should use it rather than to short circuit it as the string instance may do other processing we'd otherwise skip.